samples/highcharts/demo/combo-data-resolution/demo.js

```diff
@@ -0,0 +1,47 @@
+Highcharts.chart('container', {
+    title: {
+        text: 'Global temperature mean anomalies from 1880 to 2016',
+        align: 'left'
+    },
+    accessibility: {
+        description: `
+            A chart of monthly mean temperature anomalies from 1880 to 2016.
+            It employs two series, one column and one line,
+            to convey the same data at different resolutions.
+            There is only one column per thirty years,
+            but the line series have a datapoint per year.
+        `
+    },
+    series: [{
+        type: 'column',
```

Comment:
The columns appear with a fixed width, not reflecting their interval. I am not sure why, but I think it's because the other series is not sorted.

Anyway, we must supply a `pointRange` of 30 years in order for this to render like we expect. I think we should also remove the padding because the 30-year period extends to the whole time span. Something like this:

```js
,
        pointRange: 30 * 365 * 24 * 36e5,
        groupPadding: 0,
        pointPadding: 0,
        pointPlacement: -0.5
```

The `pointPlacement` places the column to the left of the value. So if x is on new-year 2020, the column will span from 1990-2020. Which leads to another issue, namely that the columns are registered 5 years prior, in 2015 etc.